accesshandler/pkg/providers/azure/ad/ad.go

```diff
@@ -0,0 +1,49 @@
+package ad
+
+import (
+	"context"
+
+	"github.com/common-fate/granted-approvals/accesshandler/pkg/genv"
+	"github.com/common-fate/granted-approvals/pkg/deploy"
+	"github.com/invopop/jsonschema"
+	"go.uber.org/zap"
+)
+
+const MSGraphBaseURL = "https://graph.microsoft.com/v1.0"
+const ADAuthorityHost = "https://login.microsoftonline.com"
+
+type Provider struct {
```

Comment:
All these properties should be un-exported "lowercase" to help prevent unintentional logging of credentials